carve.lic

```diff
@@ -0,0 +1,118 @@
+=begin
+  Suggestions and contributions are welcome: https://github.com/rpherbig/dr-scripts
+=end
+
+custom_require(%w(common common-crafting))
+
+class Carve
+  include DRC
+  include DRCC
+
+  def initialize(args:[])
+    settings = get_settings
+    @bag = settings.crafting_container('backpack')
+    @finish, @type, @chapter, @recipe_name, @material, @noun = args
```

Comment:
We should do a check that the correct number of args were passed in. I'd hate to see it get partway through a craft and then fail on a missing arg. `Forge` does this, but `shape` and `sew` do not.

Don't worry about it as part of this PR, but we'll want to look into it later.